css/report/iframe-answer.less

```diff
@@ -5,6 +5,16 @@
     width: 100%;
     display: flex;
     flex-direction: column;
+
+    &.scaled {
+      height: 100%;
+      max-height: 100%;
+    }
+  }
+  &.scaled {
+    height: auto;
+    max-height: 300px;
+    overflow: hidden;
```

Comment:
@emcelroy can you describe how this is changing the behavior from before?
I normally skip over CSS changes, but here I wondering how it will effect the height when the interactive is requesting its own height. Some interactives might not look good if you give them the full height instead of what they ask for.

Reply:
@scytacki These rules are applied to a wrapper for the iframe and its header. They're supposed to allow the interactive to use the full amount of space in the column (if it needs it) without overflowing it. 

<img width="953" alt="image" src="https://user-images.githubusercontent.com/367459/156020514-f8c112f1-26b5-4b00-9647-6c9dbc7e06ed.png">

Without the rules, the interactive can appear cropped. For example:

<img width="416" alt="image" src="https://user-images.githubusercontent.com/367459/156024109-6bc64ead-da87-44e1-95f0-8798aadd9541.png">

The rules don't affect the height of the actual interactive or it's iframe, AFAIK.

Does that seem reasonable? Or is it still possible that that could make things look bad? I did try this out with a variety of drag-and-drop interactives with different dimensions, aspect ratios, and aspect ratio settings. It seemed to work pretty well with them all, but it's possible I missed some combination that would've shown how this could be a problem.